data/wicked/static_address/ifroute-eth0

```diff
@@ -0,0 +1,2 @@
+default  10.0.2.2         -  -
+::/0     fd00:cafe:babe::1  -  -
```

Comment:
cafe:babe ? XD

Reply:
:) I just copy/pasted this from here https://github.com/openSUSE/wicked-testsuite/blob/master/test-files/static-addresses/ifcfg-eth0 
but I also notice this :) but as far as I now it is valid IPv6 :)